WordPress/Classes/ViewRelated/Blog/SharingViewController.m

```diff
@@ -44,12 +45,12 @@ - (void)viewDidLoad
 {
     [super viewDidLoad];
 
-    self.parentViewController.navigationItem.title = NSLocalizedString(@"Sharing", @"Title for blog detail sharing screen.");
+    self.navigationItem.title = NSLocalizedString(@"Sharing", @"Title for blog detail sharing screen.");
```

Comment:
This is a revert from when it was being wrapped in a [`JetpackBannerWrapperViewController`](https://github.com/wordpress-mobile/WordPress-iOS/pull/19106/files).